### DISCUSSION TITLE
24/7 recording on substream? events on mainstream

### DISCUSSION THREAD

**Discussion by jubjubrsx** [blakeblackshear/frigate#7283] [Ask A Question]
Recently switched over to Frigate, so far liking it alot. Even got me a Coral m.2 on the way. Although I have one question I cant seem to figure out in the config, and brave/google has been no help. Basically I'd like to record 24/7 on my substream (640x480) but the events record on the main stream (4k) everytime I try to add in the record under roles it says I can only have each role once.

**Comment by NickM-27** [comment]
This is not supported currently, there is a pinned feature request.

**Reply by jubjubrsx** [reply]
Thank you for the reply, guess I missed that original post.

**Reply by NickM-27** [reply]
You can see it at https://github.com/blakeblackshear/frigate/issues/2652

**Comment by ministryofsillywalks** [comment]
Is there any workaround to achieve this?
I really would like 24/7 substream recording but events on main stream.

**Reply by jubjubrsx** [reply]
As a work around, I just added a second camera on the sub stream set to record... So I have 6 physical cameras but in frigate I sure 12 cameras
